plugins/kafka/replica-topics-check.rb

```diff
@@ -0,0 +1,44 @@
+require 'sensu-plugin/check/cli'
+
+class ReplicaTopicCheck < Sensu::Plugin::Check::CLI
+	option :zookeeper,
+		description: 'ZooKeeper connect string',
+		short:       '-z NAME',
+		long:        '--zookeeper NAME',
+		default:     'kaf-01:2181'
+
+	option :kafka_home,
+		description: 'Kafka home',
+		short:       '-k NAME',
+		long:        '--kafka-home NAME',
+		default:     '/opt/kafka'
+
+
+	# read the output of a command
+	# @param cmd [String] the command to read the output from
+	def check_output(cmd)
+		lines = 0;
+		IO.popen(cmd + ' 2>&1') do |child|
+			lines = child.readlines().size
+		end
+
+		return lines
+	end
+
+	def run
+		kafka_topics_script = "#{config[:kafka_home]}/bin/kafka-topics.sh --zookeeper #{config[:zookeeper]} --describe"
+		unknown "Can not find #{kafka_topics_script}" unless File.exist?(kafka_topics_script)	
```

Comment:
This condition will always fail because it will test that `/opt/kafka/bin/kafka-topics.sh --zookeeper localhost-01:2181 --describe` file exists